assets/samples/dynakube/v1beta3/applicationMonitoring.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: dynatrace.com/v1beta3
+kind: DynaKube
+metadata:
+  name: dynakube
+  namespace: dynatrace
+spec:
+  apiUrl: https://ENVIRONMENTID.live.dynatrace.com/api
+
+  # Link to api reference for further information: https://docs.dynatrace.com/docs/ingest-from/setup-on-k8s/reference/dynakube-parameters#tabgroup--apiversions--v1beta2
```

Comment:
this links to `v1beta2`, should link to `v1beta3`. I only linked to old api in my example because the new one isn't public yet

Reply:
should I remove it for now?

Reply:
no, maybe just link to the page itself instead of the subheading

Reply:
done https://github.com/Dynatrace/dynatrace-operator/pull/4160/commits/232423ff5382b8502a544ea25f32d512afd24897